ledger/voters_test.go

```diff
@@ -122,9 +122,8 @@ func TestVoterTrackerDeleteVotersAfterStateproofConfirmed(t *testing.T) {
 	conf := config.GetDefaultLocal()
 	// To cause all blocks to be committed, for easier processing by the voters tracker.
 	conf.MaxAcctLookback = 0
-	au, ao := newAcctUpdates(t, ml, conf)
-	defer au.close()
-	defer ao.close()
+	_, ao := newAcctUpdates(t, ml, conf)
+	// au and ao are closed via ml.Close() -> ml.trackers.close()
```

Comment:
`au` is no longer a variable.
```suggestion
	// accountUpdates and onlineAccounts are closed via: ml.Close() -> ml.trackers.close()
```